terrarun/server/__init__.py

```diff
@@ -816,7 +808,7 @@ def check_permissions_post(self, organisation_name, current_user, current_job, *
         """Check permissions"""
         organisation = Organisation.get_by_name_id(organisation_name)
         if not organisation:
-            print('NOT ORG FOND')
+            logger.error('Organisation "%s" not found.', organisation_name)
```

Comment:
Just because Yoda isn't fond of an organisation - doesn't mean it doesn't exist!